elastic/elasticsearch-sql-odbc
# Integration testing

This PR adds a Python application that performs integration testing against Elasticsearch:
- fetch an Elasticsearch distribution matching the driver version,
- unzip it, change config, start it,
- enable XPack trial mode and authentication,
- install driver to test into host OS;
- download test CSVs and index them;
- run tests through pyodbc module, using the driver.

Except for the driver installation part, the application should be OS-independent.

Currently the tests consist of simple SELECTs against the indexed CSV, these will be supplemented in time.

The PR also adds a new parameter to the `build.bat` instructing it to perform all CI tests: unit tests against Debug builds and integration tests against Release builds, for both x86 and x64 architectures.
